source/qb64pe.bas

```diff
@@ -24708,6 +24708,67 @@ SUB PreParse (e$)
     DIM f AS _FLOAT
     STATIC TotalPrefixedPP_TypeMod AS LONG, TotalPP_TypeMod AS LONG
 
+    DO 'convert &H values to decimal values to prevent errors
+        l = INSTR(l, UCASE$(e$), "&H")
+        IF l THEN
+            FOR l1 = l + 2 TO LEN(e$)
+                SELECT CASE UCASE$(MID$(e$, l1, 1))
+                    CASE "0" TO "9"
+                    CASE "A" TO "F"
+                    CASE ELSE:
+                        PRINT UCASE$(MID$(e$, l1, 1))
```

Comment:
We probably don't want to `PRINT` :D

Reply:
Debugging statement missed.  LOL.  Good catch.  :)

Reply:
Fixed